modules/t/transcriptVariationAdaptor.t

```diff
@@ -350,4 +352,46 @@ my $msc_2_expected = 'test_consequence_a';
 my $msc_2 = $trvar_msc->most_severe_OverlapConsequence();
 is($msc_2->SO_term, $msc_2_expected, 'tv - most_severe_OverlapConsequence - same rank');
 
+
+## RefSeq Mismatch Testing
+
+my $tr = $tr_ad_of->fetch_by_stable_id('NM_001270408.1');
+my $sl_refseq = $s_ad->fetch_by_region('chromosome', 21);
+$vf_ad->db->shift_hgvs_variants_3prime(0);
+
+my $vf_refseq = Bio::EnsEMBL::Variation::VariationFeature->new
+  (-seq_region_name => 21,
+   -start => 25700000,
+   -end   => 25700000,
+   -slice => $sl_refseq,
+   -strand => 1,
+   -variation_name => 'refseq_test',
+   -allele_string => 'G/A',
+   -adaptor     => $vf_ad
+);
+
+$tv = Bio::EnsEMBL::Variation::TranscriptVariation->new(
+    -transcript     => $tr,
+    -variation_feature  => $vf_refseq,
+    -adaptor      => $trv_ad,
+    -no_shift     => 1,
+);
+
+## Setting CDS coordinates allows us to set up niche case where mismatches are calculated
+my @tvas = @{ $tv->get_all_alternate_TranscriptVariationAlleles };
+$tvas[0]->{pre_consequence_predicates}->{exon} = 1;
+
+$tv->cds_start(1234);
+$tv->cds_end(1233);
+$tv->cdna_start(1000);
+$tv->transcript->{cdna_coding_start} = 234;
+
+## Coordinate within HGVS matches cds_start as these values take mismatch into account 
+ok($tv->hgvs_transcript->{A} eq 'NM_001270408.1:c.1234N>A', 'Refseq HGVS mismatch calculated');
```

Comment:
Why is the reported hgvs containing 'N'? Could a comment be added or the PR example used here instead: `NC_000012.11:g.103249104C>A` leading to correct `NM_000277.3:c.516G>T`?

Reply:
The test database does not include sequence data for the region the test is looking in. The N isn't relevant to what we're testing for, so I just left it.